apps/wallet/src/ui/app/components/transactions-card/index.tsx

```diff
@@ -46,22 +47,29 @@ export function TransactionCard({ txn, address }: TransactionCardProps) {
     // Get the balance changes for the transaction and the amount
     const balanceChanges = getBalanceChangeSummary(txn, recognizedPackagesList);
 
-    function getAmount(tx: IotaTransactionBlockResponse) {
-        if ((isTimelockedStaking || isTimelockedUnstaking) && tx.events) {
-            return getTransactionAmountForTimelocked(
-                tx.events,
+    const [balance, coinType] = (() => {
+        if ((isTimelockedStaking || isTimelockedUnstaking) && txn.events) {
```

Comment:
If the tx if of type migration we should also always render the iota balance balance